src/MiniZinc.jl

```diff
@@ -69,12 +69,30 @@ function MOI.supports_constraint(
     return true
 end
 
-MOI.supports(::Model, ::MOI.NLPBlock) = true
+_PREFIX_OPS =
+    Dict(:(!) => "not", :∃ => "exists", :∀ => "forall", :count => "count")
```

Comment:
```suggestion
    Dict(:(!) => "not", Symbol("∃") => "exists", :∀ => "forall", :count => "count")
```
The Symbol `:∃` isn't valid Julia syntax.